plugins/apel/src/auditor_apel_plugin/republish.py

```diff
@@ -27,13 +28,22 @@ def run(config, client, args):
     site = args.site
 
     begin_month = datetime(year, month, 1).replace(tzinfo=timezone.utc)
+    if month == 12:
+        end_month = datetime(year + 1, 1, 1).replace(tzinfo=timezone.utc)
+    else:
+        end_month = datetime(year, month + 1, 1).replace(tzinfo=timezone.utc)
```

Comment:
Just a comment: In the future we could use `dateutil.relativedelta` for this, but right now there is no version for python 3.10 and 3.11 ...